pkg/pod/status_test.go

```diff
@@ -629,7 +631,7 @@ func TestMakeTaskRunStatus(t *testing.T) {
 				}},
 				TaskRunResults: []v1beta1.TaskRunResult{{
 					Name:  "resultName",
-					Value: "resultValue",
+					Value: *v1beta1.NewArrayOrString("resultValue"),
```

Comment:
This should be a value not a pointer. Same beflow

Reply:
unit tests are passing, check this for last commit or all recent runs https://tekton-releases.appspot.com/build/tekton-prow/pr-logs/pull/tektoncd_pipeline/4818/pull-tekton-pipeline-unit-tests/1522392776887504899/

I'm failing build tests because of linting. 😄 

This one is tricky because I'm using a func to create arrayorstring, it returns a pointer, so actually here is a value not pointer

Reply:
Ah I see. This thread can be closed